spec/integrations/shared/base_render_examples.rb

```diff
@@ -287,6 +287,37 @@ def self.unless_method(_object, _options)
     end
   end
 
+  context 'Given blueprint has ::root' do
+    let(:result) { '{"root":{"id":' + obj_id + ',"position_and_company":"Manager at Procore"}}' }
+    let(:blueprint) { blueprint_with_block }
+    it('returns json with a root') do
+      expect(blueprint.render(obj, root: :root)).to eq(result)
+    end
+  end
+
+  context 'Given blueprint has ::meta' do
+    let(:result) { '{"root":{"id":' + obj_id + ',"position_and_company":"Manager at Procore"},"meta":"meta_value"}' }
+    let(:blueprint) { blueprint_with_block }
+    it('returns json with a root') do
+      expect(blueprint.render(obj, root: :root, meta: 'meta_value')).to eq(result)
+    end
+  end
+
+  context 'Given blueprint has ::meta without root' do
+    let(:result) { '{"id":' + obj_id + ',"position_and_company":"Manager at Procore"}' }
+    let(:blueprint) { blueprint_with_block }
+    it('returns json with a root') do
```

Comment:
Is this description accurate? Shouldn't this be `returns json without root and meta` since a root is not provided?
Should this just drop the meta or should it throw a Blueprinter error since a root was not specified?

Reply:
The implementers would clearly know from responses generated. Moreover, we have explicitly called this out in both the readme and the documentation/comments in the methods. Any deviations in future from this behaviour would be caught by this Spec.

Reply:
I don't think we should rely on the implementers noticing something isn't showing up. I would not be satisfied with the answer "read the documentation better" when something is failing silently on me.